repository/packages/azure/psql/0.18.0.yml

```diff
@@ -0,0 +1,248 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  creationTimestamp: null
+  name: psql.azure.references.services.apps.tanzu.vmware.com.0.18.0
+spec:
+  refName: psql.azure.references.services.apps.tanzu.vmware.com
+  releasedAt: "2022-09-16T08:54:15Z"
+  template:
+    spec:
+      deploy:
+      - kapp: {}
+      fetch:
+      - imgpkgBundle:
+          image: ghcr.io/vmware-tanzu-labs/trp-azure-psql@sha256:df5786e7558e1de00dce5d099ba1dd52f7acc52c38010eb88a14b35a5c8d67de
```

Comment:
_issue_: Please use the the namespace `vmware-tanzu/tanzu-application-platform-reference-service-packages` and image name similar to the packages' refName.

Reply:
Agreed. We're waiting to get an OKE on the package itself before we publish it and then update this "version" file.